msteams-platform/messaging-extensions/create-api-message-extension.md

```diff
@@ -113,7 +113,9 @@ To test your API-based message extension created in the Developer Portal for Tea
 # [Visual Studio Code](#tab/visual-studio-code)
 
 > [!NOTE]
-> Teams Toolkit support for API-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+> * Teams Toolkit support for API-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+>
+> * Teams Toolkit only supports OpenAPI Specification version 3.0.1.
 
```

Comment:
Is this accurate? @SLdragon

Reply:
Teams Toolkit support OpenAPI spec version 2.0, 3.0.x, and 2.0 will be automatically converted to 3.0.x

Reply:
Updated.